modules/core/05-port/client/cli/query.go

```diff
@@ -0,0 +1,54 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/version"
+
+	channeltypes "github.com/cosmos/ibc-go/modules/core/04-channel/types"
+	"github.com/cosmos/ibc-go/modules/core/05-port/types"
+	host "github.com/cosmos/ibc-go/modules/core/24-host"
+)
+
+// GetCmdQueryPorts defines the command to query a port
```

Comment:
I'm unsure if a CLI cmd is really useful here or not. Thoughts?

Reply:
Hmm, I think we could probably remove the CLI command for this. I can't think of any reason to use it? I'm in favor of having as little code to maintain as possible if you are. :D 

Can you think of any advantage for having the CLI command as well? @colin-axner

Reply:
I am also in favor of keeping a very lean cli. I pushed pretty hard to remove all the handshake cli function :smile: 

I do see benefit of adding a cli function to query ports that are currently binded and/or a cli to query what module belongs to a port. So if I query `transfer` I should get back `ics20`. The app negotiation part definitely does not need a cli. The cli cmds for port could be addressed in a pr for #112 (which this partially addresses)

I'll leave my comments down below as I still need to think about the proposed gRPCs and their usage. Would be beneficial to get input from the Hermes team

Reply:
Sounds good. I can take on that PR for #112 following on from the ICA work if you like.

Reply:
That'd be fantastic honestly. This was one of the first feature requests we had after launching the initial IBC module